values-development.yaml

```diff
@@ -85,6 +55,15 @@ clusterGroup:
         jsonPointers:
         - /imagePullSecrets
         - /secrets
+      overrides:
+      - name: devel.image
+        value: chat-client:latest
+      - name: devel.npmbase
+        value: quay.io/hybridcloudpatterns/ubi-npm
+      - name: devel.namespace
+        value: devsecops-ci
+      - name: devel.roxCTLImage
+        value: registry.access.redhat.com/advanced-cluster-security/rhacs-roxctl-rhel8:3.71.0-5
```

Comment:
Is this mechanism preferred over the below?

```
devel: 
   image: chat-client:latest
   npmbase: quay.io/hybridcloudpatterns/ubi-npm
   namespace: devsecops-ci
   roxCTLImage: registry.access.redhat.com/advanced-cluster-security/rhacs-roxctl-rhel8:3.71.0-5
```

/cc @mhjacks @mbaldessari

Reply:
Obviously both will work, but we likely want to pick one consistently across the patterns